### PR TITLE
Muting expected error logging during tests.

### PR DESCRIPTION
**Pull request by cschroed-usgs**

### CODE REVIEW

File: test/js/portal_ui/queryServiceSpec.js [hunk 1]
```diff
@@ -66,12 +66,21 @@ describe('Tests for queryService', function() {
 		});
 
 		it('Expects a failed response to reject the promise', function() {
-			PORTAL.queryServices.fetchQueryCounts('Result', testQuery, ['NWIS', 'STORET']).done(successSpy).fail(errorSpy);
```
Comment (cschroed-usgs) [inline test/js/portal_ui/queryServiceSpec.js]: These removed lines were all pasted verbatim into the `try` block

**Comment by cschroed-usgs** [comment]
Changes this test output:
```
02 03 2017 16:44:47.254:WARN [web-server]: 404: /base/wqp/portal_ui/static/img/expand.png
.......
ERROR: 'Unable to contact the WQP services: error'
................................................................................
.......................
Firefox 50.0.0 (Linux 0.0.0): Executed 243 of 244 (skipped 1) SUCCESS (3.663 secs / 3.158 secs)

```
To this test output:
```
02 03 2017 16:47:38.934:WARN [web-server]: 404: /base/wqp/portal_ui/static/img/expand.png
................
...
Firefox 50.0.0 (Linux 0.0.0): Executed 243 of 244 (skipped 1) SUCCESS (3.715 secs / 3.208 secs)
```

File: test/js/portal_ui/queryServiceSpec.js [hunk 2]
```diff
@@ -66,12 +66,21 @@ describe('Tests for queryService', function() {
...
-			fakeServer.respondWith([404, {"Content-Type" : 'text/html'}, 'Not found']);
-			fakeServer.respond();
+			//We expect the test to log error messages, but we don't want them in the test output
+			//so we will record the initial logging level, mute logging, then finally restore the logging level
+			var initialLevel = log.getLevel();
+			
+			try{
```
Comment (mbucknell) [inline test/js/portal_ui/queryServiceSpec.js]: Consider putting the log.disableAll in the beforeEach and the log.setLevel in afterEach. I think this is a better pattern than using a try-finally block.

Comment (cschroed-usgs) [inline test/js/portal_ui/queryServiceSpec.js]: Good idea! Will do.

**Pull request merged**